Makefile

```diff
@@ -1,6 +1,7 @@
 APP_NAME = reconciler
 IMG_NAME := $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME)
 TAG := $(DOCKER_TAG)
+COMPONENTS := $(shell (cd pkg/reconciler/instances && printf -- '%s' */ | tr '/' ',' | sed -e "s/^example,//" -e "s/,$$//" && cd ../../..))
```

Comment:
wow, that looks quite hard to read :) not sure if the following command is simpler to read, but what about:

`find pkg/reconciler/instances -d 1 -type d -not -path '*/example' -exec sh -c 'echo $(basename {})' \;`

Reply:
This works better  - without line breaks in the output:

`find pkg/reconciler/instances -d 1 -type d -not -path '*/example' -execdir echo -n '{} ' \;`